.pre-commit-config.yaml

```
@@ -0,0 +1,30 @@
+exclude: |
+  (?x)^(
+    .*third_party/.*|
+    .*laf/.*|
+  )
+
+repos:
```

Some extra hooks to add:
```suggestion
repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.6.0
    hooks:
      - id: end-of-file-fixer
      - id: trailing-whitespace

  - repo: https://github.com/Lucas-C/pre-commit-hooks
    rev: v1.5.5
    hooks:
      - id: remove-tabs
        args: [--whitespaces-count, '8']

```